### PR TITLE
Expand acknowledgements

### PR DESCRIPTION
**Pull request by bemasc**

### CODE REVIEW

**Comment by bemasc** [comment]
@enygren @MikeBishop Please note anyone else who we ought to include in this list.

@pieterlexis @wtoorop @puneetsood Please let us know if you would prefer not to be included.

**Comment by wtoorop** [comment]
Thanks! Appreciated.
Tom Carpay (@TCY16 ) should be listed there too.
Tom did the commits with the failure-cases.

File: draft-ietf-dnsop-svcb-https.md [hunk 1]
```diff
@@ -1592,7 +1592,9 @@ Thank you to Ian Swett, Ralf Weber, Jon Reed,
 Martin Thomson, Lucas Pardue, Ilari Liusvaara,
 Tim Wicinski, Tommy Pauly, Chris Wood, David Benjamin,
 Mark Andrews, Emily Stark, Eric Orth, Kyle Rose,
-Craig Taylor, Dan McArdle, Brian Dickson, and others for their feedback
+Craig Taylor, Dan McArdle, Brian Dickson,
+Willem Toorop, Pieter Lexis, Puneet Sood,
+and many others for their feedback
```
Comment (wtoorop) [inline draft-ietf-dnsop-svcb-https.md:1597]:
```suggestion
Tom Carpay, and many others for their feedback
```

**Comment by enygren** [comment]
Others?
@rs ( Olivier Poitrey )
@mmuhaime ( Mashooq Muhaimen )
@jdreed ( Jonathan Reed )

Not a comprehensive check, but some who I noticed as missing but who have provided valuable feedback on recent PRs.

**Comment by bemasc** [comment]
OK, added.

**Comment by rs** [comment]
Ok too

**Pull request merged**